build/build.ps1

```diff
@@ -21,12 +21,12 @@ function Build-One {
     );
 
     Write-Host "##[info]Building $project ($action)..."
-    dotnet $action (Join-Path $PSScriptRoot $project) `
-        -c $Env:BUILD_CONFIGURATION `
-        -v $Env:BUILD_VERBOSITY `
-        /property:DefineConstants=$Env:ASSEMBLY_CONSTANTS `
-        /property:Version=$Env:ASSEMBLY_VERSION `
-        /property:QsharpDocsOutputPath=$Env:DOCS_OUTDIR
+    Invoke-Expression ("dotnet $action $(Join-Path $PSScriptRoot $project) " +
+        "-c $Env:BUILD_CONFIGURATION " +
+        "-v $Env:BUILD_VERBOSITY " +
+        "$(if ($Env:ASSEMBLY_CONSTANTS){"/property:DefineConstants=$Env:ASSEMBLY_CONSTANTS"}) " +
+        "/property:Version=$Env:ASSEMBLY_VERSION " +
+        "/property:QsharpDocsOutputPath=$Env:DOCS_OUTDIR")
```

Comment:
I have been trying to look for a way to do this in PowerShell that doesn't involve string concatenation, but I have failed. :) I guess it might be the best way here.

Reply:
Sadly, it is also the best I could come up with too...